modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -0,0 +1,58 @@
+= Accessing the console
+
+The {app-security-service} Console is viewed by opening the console URL in a browser. This URL is seen in the list of available services in the Mobile Developer Console (MDC) after the {app-security-service} Service has been provisioned.
+
+= Console overview
```

Comment:
I think this overview should mention the most important thing about this application - the fact that the user can update the disabled status and disabled message for each app.